src/_variables.scss

```diff
@@ -0,0 +1,22 @@
+$font__family--main: 'Roboto', sans-serif;
+
+$font__size--headings-biggest: 72px;
+$font__size--headings-bigger: 35px;
+$font__size--headings-big: 22px;
+$font__size--headings-little: 19px;
+$font__size--headings-less: 15px;
+$font__size--headings-least: 12px;
+$font__size--headings-extrasmall: 10px;
+
+$color--light-text: #ffffff;
+$color--dark-text: #000000;
+$color--darker-text: #333333; 
+
+
+$color--primary: #bb0a30;
+$color--light: #FFFFFF;
```

Comment:
HEX lowercase?

Reply:
CSS values for colors are not case-sensitive.

Reply:
You are right, they are case-insensitive.
But you should use uppercase or lowercase consistently in your code.

Reply:
Edited to lowercase .